src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/markduplicates/PairedEnds.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.hellbender.tools.dataflow.transforms.markduplicates;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+/**
+ * Struct-like class to store information about the paired reads for mark duplicates.
+ */
+public final class PairedEnds {
```

Comment:
Nope, the coder needs to be able to see it, and we decided that coders should live in a coder package.